### ISSUE TITLE
Weird shadows behavior in Lake View Hotel

### ISSUE THREAD

**Issue by Fabiusco** [elishacloud/Silent-Hill-2-Enhancements#224]
First of all I want to thank you for the effort you are putting into this amazing project. 
The results is absolutely stunning and it improve with every release. You gave me another reason to play this masterpiece over and over. 
Anyway, today I noticed some issues with some shadow in Lake View Hotel. 
In the outside area, James shadow is still “multi-layered“. 
Other than that, I noticed weird shadow behaviors when you go to the second floor by using the stairs in the first room when you enter the hotel. Just after the fade-in animation, shadows flick for a second. Something similar also happens in the second floor corridor (the one with room 202). The shadow casted by the wall turning right, just after the entrance door, acts weirdly. 
I hope this could be useful!

**Comment by Polymega** [comment]
We're glad you're enjoying the project! 

> In the outside area [of Lakeview Hotel], James shadow is still “multi-layered“.

Yes, this is normal behavior. Whenever James is outdoors (during the daytime), he will have a "multi-layered" shadow except for the following areas: Observation Deck, Angela apartment room, and both Eddie boss fight rooms. [More info on that here.
](https://youtu.be/uJi0_kJbCoU?t=307)
Here is a PS2 screenshot for comparison:
![image](https://user-images.githubusercontent.com/27395260/74883208-71c47680-533e-11ea-9dce-ea6264124089.png)


> Other than that, I noticed weird shadow behaviors [...]

This is normal shadow behavior for the game. :) SH2's shadow programming was amazing for its time, but it's not perfect. 

Here is the same behavior on the PS2 version, for comparison: https://youtu.be/APW9Rpgj5FI

Simply put: For better and worse, the new shadow work in our project is 1:1 accurate to the console versions.

**Comment by elishacloud** [comment]
Closing this out.
